### DISCUSSION TITLE
Driveway Parked Car Alerts

### DISCUSSION THREAD

**Discussion by spikked27** [blakeblackshear/frigate#13731] [Ask A Question]
Driveway Parked Car alert is driving me crazy, I have a zone called driveway that excludes cars and car object masks everywhere except the entrance to the driveway but I'm still getting alerts every few minutes, any tips?  Only way I was able to resolve this was putting a car object mask over the whole driveway but that doesn't seem like the correct solution
![Screenshot_20240913_190521_Chrome Beta](https://github.com/user-attachments/assets/9b528649-95f7-4f2b-904e-7706e8bfa3ca)
![Screenshot_20240913_222517_Chrome Beta](https://github.com/user-attachments/assets/f53760f2-32de-4c91-953d-04a06d0921d4)

**Comment by NickM-27** [comment]
You should submit a support request so we can see all important information. Also, a video of the debug view would be helpful
